frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -55,8 +64,8 @@ export const dashboardLogic = kea<dashboardLogicType<DashboardLogicProps>>({
         updateLayouts: (layouts: Layouts) => ({ layouts }),
         updateContainerWidth: (containerWidth: number, columns: number) => ({ containerWidth, columns }),
         saveLayouts: true,
-        updateItemColor: (id: number, color: string) => ({ id, color }),
-        setDiveDashboard: (id: number, dive_dashboard: number | null) => ({ id, dive_dashboard }),
+        updateItemColor: (insightId: number, color: string) => ({ insightId, color }),
+        setDiveDashboard: (insightId: number, dive_dashboard: number | null) => ({ insightId, dive_dashboard }),
```

Comment:
Does it make sense to also use shortId's here? I think it could be confusing if dashboardLogic actions requires different id parameter types.

Reply:
This is because we're directly passing this to an API update call. Since it's an internal function, with the correct type, I think it doesn't make a difference. It avoids another call to `async getInsightId()`.